release/nightly_tests/stress_tests/state_api_app_config.yaml

```diff
@@ -6,7 +6,7 @@ python:
   pip_packages:
     - terminado
     - boto3
-    - cython==0.29.26
```

Comment:
I am removing these from this PR, let's not touch this. 
https://github.com/ray-project/ray/pull/31339